error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -440,6 +440,8 @@ Duration testStepVerifierLastStepVerifyError() {
   ImmutableSet<Duration> testStepVerifierLastStepVerifyErrorClass() {
     return ImmutableSet.of(
         StepVerifier.create(Mono.empty()).expectError(IllegalArgumentException.class).verify(),
+        StepVerifier.create(Mono.empty())
+            .verifyErrorMatches(IllegalStateException.class::isInstance),
         StepVerifier.create(Mono.empty())
             .verifyErrorSatisfies(t -> assertThat(t).isInstanceOf(IllegalStateException.class)));
```

Comment:
For consistency with other test cases, let's vary this exception type. I see that elsewhere in this class we use `AssertionError` as a third type.